canonical-websites/jaas.ai
# Remove the charmers page

## Done

- Remove the Charmers page.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Visit /community/charmers
- You should get a 404.
- Visit /community and the "Community" section at the top should have link that says "Join us in Discourse ›"

## Details

- Fixes: #101.
